### DISCUSSION TITLE
Enable verbose output/debugging?

### DISCUSSION THREAD

**Discussion by synergiator** [gitpython-developers/GitPython#1163] [Q&A] [answered]
I've a scenario where GitPython seems to idle shortly after starting. Is it possible to pass over debug/verbose output (of git?) ?

**Comment by Byron** [comment, marked as answer]
I believe the question is answered sufficiently well by the [respective section in the documentation](https://gitpython.readthedocs.io/en/stable/tutorial.html#git-command-debugging-and-customization).

Closing this issue, please feel free to keep commenting in case of follow-up questions.
